user/src/routes/routes.ts

```diff
@@ -0,0 +1,26 @@
+import { Application, Router } from 'express';
+
+export const initializeUserRoutes = (app: Application) => {
+    const userRouter = Router();
+    app.use('/user', userRouter);
+
+    /* gets a user */
+    userRouter.get('/', async (req, res) => {
+        res.status(200).send("Working user get route")
```

Comment:
Minor change, but can you change these strings to single quotes? I'm just getting tslint errors in my editor!